pkg/operator/ceph/cluster/config.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
nit: How about just adding the contents of this file to monstore.go? It all seems related.

Reply:
Good point, I moved it to the `monstore.go` file.